CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Added
 
+- #951, Refactor createRawTransaction coin hour distribution
```

Comment:
can you describe it a little more, this will be relevant to the exchanges and they read the changelog when upgrading